src/main/resources/assets/viafabric/lang/tr_tr.json

```diff
@@ -0,0 +1,14 @@
+{
+  "gui.protocol_version_field.name": "Protocol Versiyonu",
```

Comment:
"gui.protocol_version_field.name": "Protokol Versiyonu",

Reply:
Ah, must’ve missed that one because they sound/look so familiar. Fixed now